paddle/fluid/inference/analysis/ir_passes/tensorrt_subgraph_pass.cc

```diff
@@ -281,6 +280,99 @@ void TensorRtSubgraphPass::CreateTensorRTOp(
           trt_engine_serialized_data);
 }
 
+void RenameAndGetOutputs(
```

Comment:
Refined here: https://github.com/PaddlePaddle/Paddle/blob/eb83abeac3c0146b921ce72d06fef2551ab3e8d8/paddle/fluid/inference/analysis/ir_passes/subgraph_util.h#L38

Reply:
fixed, thx for reminding